examples/sample_measurement/measurement.py

```diff
@@ -60,6 +61,7 @@ class Color(Enum):
     "Protobuf Enum out", nims.DataType.Enum, enum_type=color_pb2.ProtobufColor
 )
 @measurement_service.output("String Array out", nims.DataType.StringArray1D)
+@measurement_service.output("Double2DArray Out", nims.DataType.Double2DArray)
```

Comment:
"When in Rome, do as the Romans do." By which I mean: look at the existing code and follow the same patterns and conventions.

```suggestion
@measurement_service.output("Double 2D Array Out", nims.DataType.Double2DArray)
```